src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.GuardedCallsTests.cs

```diff
@@ -86,8 +86,8 @@ void Api()
     }
 }" + MockAttributesCsSource + MockOperatingSystemApiSource;
 
-            await VerifyAnalyzerAsyncCs(source, s_msBuildPlatforms, VerifyCS.Diagnostic(PlatformCompatibilityAnalyzer.SupportedOsVersionRule).WithLocation(15, 9)
-                .WithMessage("'Test.Api()' is unsupported on 'windows'"));
```

Comment:
@mavasani Would it be worth banning the `WithMessage` API? It seems that we favor using `WithArguments` to avoid translation issues.